recipe/recipe.yaml

```diff
@@ -0,0 +1,228 @@
+        - typing_extensions
+        - if: unix
+          then:
+            - ${{ compiler('c') }}
+            - ${{ compiler('cxx') }}
+        - if: linux
+          then: ${{ compiler('fortran') }}
+        - pkg-config
+
+    script:
+      interpreter: python
+      content:
+        - import numpy, sys
+        - 'tests_to_skip = "_not_a_real_test"'
+        - if: build_platform != target_platform
+          then:
+            # there are some tests that cannot really work in emulation, see e.g. numpy/numpy#20445
+            - 'tests_to_skip += " or Test_ARM_Features"'
+            - 'tests_to_skip += " or Test_POWER_Features"'
+            # test_new_policy reruns part of test suite; including a CPU feature test that fails in emulation
+            - 'tests_to_skip += " or test_new_policy"'
+        - if: aarch64
+          then:
+            # emulation problems (apparently) on aarch
+            - 'tests_to_skip += " or (test_basic_property and float32)"'
+            - 'tests_to_skip += " or test_generalized_herm_cases"'
+            - 'tests_to_skip += " or test_generalized_nonsq_cases"'
+            - 'tests_to_skip += " or test_generalized_sq_cases"'
+            - 'tests_to_skip += " or test_herm_cases or test_sq_cases"'
+            - 'tests_to_skip += " or test_nonsq_cases"'
+        - if: osx
+          then:
+            # https://github.com/numpy/numpy/issues/27045
+            - 'tests_to_skip += " or (test_regression and test_gh25784)"'
+        - if: ppc64le
+          then:
+            # new test failures for 2.2.4; possibly emulation-related; see
+            # https://github.com/numpy/numpy/issues/28548
+            - 'tests_to_skip += " or test_einsum_sums_float32"'
+            - 'tests_to_skip += " or test_ufunc_noncontiguous[matvec]"'
+            - 'tests_to_skip += " or test_accelerate_framework_sgemv_fix "'
+        # numpy.test will show SIMD features of agent (in the past, there have been
+        # failures that occured depending on presence/absence of e.g. AVX512);
+        # for signature of numpy.test see the following (note default: label='fast'),
+        # https://github.com/numpy/numpy/blob/maintenance/1.22.x/numpy/_pytesttester.py#L81-L82
+        - 'param = {"verbose": 1, "label": "full", "tests": None}'
+        - 'extra_argv = ["-k", f"not ({tests_to_skip})", "-nauto", "--timeout=3000", "--durations=50", "--maxfail=100"]'
+        - 'print(f"running with args: kwargs={param}, extra_argv={extra_argv}")'
+        - 'sys.exit(not numpy.test(**param, extra_argv=extra_argv))'
+
+about:
+  license: BSD-3-Clause
+  license_file: LICENSE.txt
+  summary: The fundamental package for scientific computing with Python.
+  homepage: http://numpy.org/
+  repository: https://github.com/numpy/numpy
+  documentation: https://numpy.org/doc/stable/
+
+extra:
+  recipe-maintainers:
+    - jakirkham
+    - msarahan
+    - pelson
+    - rgommers
+    - ocefpaf
+    - isuruf
+    - xhochy
+    - h-vetinari
```

Comment:
Also, let's alphabetize that list in the rewrite commit.

Reply:
done